docs/pages/repo/docs/getting-started/create-new.mdx

```diff
@@ -509,7 +509,8 @@ Take a look at `turbo.json`:
 {
   "pipeline": {
     "dev": {
-      "cache": false
+      "cache": false,
```

Comment:
We should explain why `persistent: true` is needed on this page (because it's an intro to Turbo)